spec/pushmi_pullyu/cli_spec.rb

```diff
@@ -272,6 +272,30 @@
 
         expect(old_options).to eq new_options
       end
+
+      it 'makes sure an entities information is readded to reddis when deposit fails' do
+        cli.parse(['-C', 'spec/fixtures/config_wrong_swift.yml'])
+        redis = Redis.new
+        redis.zadd(PushmiPullyu.options[:queue_name], 10,
+                   '{"uuid": "123e4567-e89b-12d3-a456-426614174000", "type": "items"}')
```

Comment:
Is this the expected form, without the hash rockets?

Reply:
my thinking is that we should be expect it to be standard json, if for whatever reason we change to another type of platform.

We need to change this in both jupiter and PMPY to get the right behaviour.